### ISSUE TITLE
unable to add SRV record for DNS_Zone_Local

### ISSUE THREAD

**Issue by chen23** [F5Networks/f5-appsvcs-extension#282]
### Environment
 * Application Services Version: 3.20.0
 * BIG-IP Version: 15.1.0.3

### Summary

unable to create an SRV record for DNS_Zone_Local.  It appears that it does recognize the format of an SRV record.
### Steps To Reproduce
1. Submit the following declaration:
```json
{
    "$schema": "https://raw.githubusercontent.com/F5Networks/f5-appsvcs-extension/master/schema/latest/as3-schema.json",
    "class": "AS3",
    "declaration": {
        "class": "ADC",
        "schemaVersion": "3.17.0",
        "id": "urn:uuid:33045210-3ab8-4636-9b2a-c98d22ab915d",
        "DNSCacheStatic": {
            "class": "Tenant",
            "Test": {
                "class": "Application",
                "template": "generic",
                "dns_cache": {
                    "class": "DNS_Cache",
                    "type": "transparent",
                    "localZones": {
                        "_etcd-server-ssl._tcp.dc1.example.com":{
                            "class":"DNS_Zone_Local",
                            "type":"static",
                            "records":["_etcd-server-ssl._tcp.dc1.example.com. 0 IN SRV 0 10 2380 master.dc1.example.com."]
                        }
                    }
                }
            }
        }
    }
}
```

2. Observe the following error response:
```json
{
    "id": "9fa8bca1-f87c-4fd3-8b3e-c109e91f3749",
    "results": [
        {
            "code": 422,
            "errors": [
                "/DNSCacheStatic/Test/dns_cache/localZones: should match format \"hostname\""
            ],
            "declarationFullId": "",
            "message": "declaration is invalid"
        }
    ],
    "declaration": {}
}
```

### Expected Behavior

Should create a record

### Actual Behavior
Errors out on valid hostname

**Comment by dstokesf5** [comment]
Thank you for reporting this issue. I have confirmed the bug and added it to our internal product backlog as AUTOTOOL-1633.

**Comment by dstokesf5** [comment]
This has been resolved as part of the 3.21.0 release.
